<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QDialog" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:rgb(206, 206, 199);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="formFrame">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>Task Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>     From</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>    Done</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>   Priority</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;</string>
        </property>
        <item>
         <property name="text">
          <string>Not Yet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yes</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>To-Do List</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : white;
font-size : 15px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : #004de6;
background-color : #004de6;
font-size : 15px;</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
        </property>
        <property name="text">
         <string>     To</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(255, 78, 78);
background-color : rgb(255, 78, 78);
font-size : 15px;</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTimeEdit" name="timeEdit_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : white;
font-size : 15px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : #004de6;
background-color : #004de6;
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : #CCD1D1;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Task</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>907</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
